app/styles/app.scss

```diff
@@ -0,0 +1,63 @@
+
+$gutter: 1rem;
+$accent: #0dc8a6;
+
+body {
+  font-family: 'Oxygen', sans-serif;
+  background: #000;
+  margin: 0;
+}
+
+.everything {
+  display: flex;
+  flex-direction: column;
+}
+
+.width-container {
+  max-width: 60rem;
+  margin: 0 auto;
+}
+
+.nav-bar {
+  background: linear-gradient(to bottom, $accent, darken($accent, 3%));
+
+  &__content {
+    padding: $gutter;
+  }
+  h2{
+    margin: 0;
+    font-family: 'Oleo Script Swash Caps', cursive;
+    font-weight: normal;
+    font-size: 1.9rem;
+    color: #fff;
+    text-shadow: 0.1rem 0.1rem 0.1rem darken($accent, 6%);
+  }
+}
+
+.grid {
```

Comment:
The grid should have negative margin to account for the first and last item's margin to make things line up with the surroundings.